### PR TITLE
Signature header should have some label

### PR DESCRIPTION
**Pull request by nyaxt**
As suggested by @irori

### CODE REVIEW

File: go/signedexchange/signer.go [hunk 1]
```diff
@@ -132,5 +133,7 @@ func (s *Signer) signatureHeaderValue(i *Input) (string, error) {
 	dateUnix := s.Date.Unix()
 	expiresUnix := s.Expires.Unix()
 
-	return fmt.Sprintf("sig=*%s; validityUrl=\"%s\"; integrity=\"%s\"; certUrl=\"%s\"; certSha256=*%s; date=%d; expires=%d", sigb64, validityUrl, integrityStr, certUrl, certSha256b64, dateUnix, expiresUnix), nil
+	return fmt.Sprintf(
+		"%s; sig=*%s; validityUrl=\"%s\"; integrity=\"%s\"; certUrl=\"%s\"; certSha256=*%s; date=%d; expires=%d",
```
Comment (hajimehoshi) [inline go/signedexchange/signer.go]: You can use `%q` instead of `\"%s\"`

**Review by hajimehoshi — CHANGES_REQUESTED** [review]

File: go/signedexchange/signer.go (hunk 1 above)
Comment (nyaxt) [reply go/signedexchange/signer.go]: done

**Comment by hajimehoshi** [comment]
Does this pass the existing tests? If not, please update that...

**Review by hajimehoshi — APPROVED** [review]

**Comment by hajimehoshi** [comment]
I'll fix the test. Thank you!

**Pull request merged**